Makefile

```diff
@@ -415,3 +415,16 @@ test-app-dev: tmp/.ekam-run
 	@cp src/sandstorm/test-app/test-app.capnp tmp/sandstorm/test-app/test-app.capnp
 	@cp src/sandstorm/test-app/*.html tmp/sandstorm/test-app
 	spk dev -Isrc -Itmp -ptmp/sandstorm/test-app/test-app.capnp:pkgdef
+
+# ====================================================================
+# meteor-testapp.spk
+
+meteor-spk-0.3.2/meteor-spk:
+	@$(call color,downloading meteor-spk)
+	@curl https://dl.sandstorm.io/meteor-spk-0.3.2.tar.xz | tar Jxf -
+
+tests/assets/meteor-testapp.spk: meteor-testapp meteor-spk-0.3.2/meteor-spk meteor-testapp/client/* meteor-testapp/server/* meteor-testapp/.meteor/*
+	@PATH="$$PWD/bin:$$PATH" && cd meteor-testapp && ../meteor-spk-0.3.2/meteor-spk pack -I../src ../tests/assets/meteor-testapp.spk
+
+meteor-testapp-clean:
+	rm -rf tests/assets/meteor-testapp.spk meteor-spk-0.3.2 meteor-testapp/.meteor-spk
```

Comment:
It occurs to me that we probably don't want to delete meteor-spk as part of `ci-clean` (which the CI runs before every build), since that's just going to force it to re-download over and over the same file which never changes. But rebuilding the .spk seems like a good idea.

Usually we don't create a separate "clean" rule for each component of the build but instead add files to the list of the existing clean rules. So maybe add meteor-spk to the main `clean` rule and the other things to the `ci-clean` rule.